[2023-11-15T11:09:02.735+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=data_ingestion_gcs_dag/run_id=scheduled__2023-11-15T10:00:00+00:00/task_id=local_to_gcs_task permission to 509
[2023-11-15T11:09:02.780+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=data_ingestion_gcs_dag/run_id=scheduled__2023-11-15T10:00:00+00:00/task_id=local_to_gcs_task permission to 509
[2023-11-15T11:09:02.815+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-11-15T10:00:00+00:00 [queued]>
[2023-11-15T11:09:02.830+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-11-15T10:00:00+00:00 [queued]>
[2023-11-15T11:09:02.831+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-15T11:09:02.850+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2023-11-15 10:00:00+00:00
[2023-11-15T11:09:02.859+0000] {standard_task_runner.py:57} INFO - Started process 1176 to run task
[2023-11-15T11:09:02.862+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2023-11-15T10:00:00+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpv0nvl5k7']
[2023-11-15T11:09:02.865+0000] {standard_task_runner.py:85} INFO - Job 58: Subtask local_to_gcs_task
[2023-11-15T11:09:02.887+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:193: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-11-15T11:09:02.932+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=data_ingestion_gcs_dag/run_id=scheduled__2023-11-15T10:00:00+00:00/task_id=local_to_gcs_task permission to 509
[2023-11-15T11:09:02.937+0000] {task_command.py:416} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-11-15T10:00:00+00:00 [running]> on host 887a0754e690
[2023-11-15T11:09:03.156+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='local_to_gcs_task' AIRFLOW_CTX_EXECUTION_DATE='2023-11-15T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-15T10:00:00+00:00'
[2023-11-15T11:09:13.845+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-15T11:09:13.857+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20231115T100000, start_date=20231115T110902, end_date=20231115T110913
[2023-11-15T11:09:13.924+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-15T11:09:13.949+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
